pkg/build/prune/resolvers.go

```diff
@@ -98,30 +98,34 @@ func (o *perBuildConfigResolver) Resolve() ([]*buildapi.Build, error) {
 	completeStates := util.NewStringSet(string(buildapi.BuildPhaseComplete))
 	failedStates := util.NewStringSet(string(buildapi.BuildPhaseFailed), string(buildapi.BuildPhaseError), string(buildapi.BuildPhaseCancelled))
 
-	results := []*buildapi.Build{}
+	prunableBuilds := []buildapi.Build{}
```

Comment:
why can't this be an array of pointers?  there's going to be a lot more copying going on now right?